pjlib/src/pjlib-test/sleep.c

```diff
@@ -103,7 +103,7 @@ static int sleep_duration_test(void)
     /* Test pj_thread_sleep() and pj_gettimeofday() */
     for (i=0, avg_diff=0, max_diff=0; i<PJ_ARRAY_SIZE(duration); ++i) {
         pj_time_val start, stop;
-        pj_uint32_t msec;
+        long msec;
```

Comment:
This will cause a different warning:
```
warning: format specifies type 'int' but the argument has type 'long' [-Wformat]
                      msec, duration[i], MAX_SLIP));
```